crates/matrix-sdk/tests/integration/event_cache.rs

```diff
@@ -1748,25 +1685,21 @@ async fn test_lazy_loading() {
     // One more chunk will be loaded from the store. This new chunk contains 5
     // items. No need to reach the network.
     {
-        let pagination_outcome = room_event_cache
-            .pagination()
-            .run_backwards(10, until_at_least_one_event)
-            .await
-            .unwrap();
+        let outcome = room_event_cache.pagination().run_backwards_until(1).await.unwrap();
```

Comment:
I was happy with `pagination_outcome` because it tells it's the outcome of the pagination 😛. Now we don't know what outcome it is about except by looking at the expression.

Reply:
The name is a bit too long, and makes statements overflow in multiple lines, when they could hold into a single one. "outcome" only refers to "pagination outcome" in the entire file. Also, each definition is at most 4 or 5 lines away from each use, so I found that having a short name is fine in this context.